cmd/root.go

```diff
@@ -242,13 +251,24 @@ func (r *Root) execute() {
 		Example: rootExamples,
 	}
 	rootCmd.CompletionOptions.DisableDefaultCmd = true
+
 	rootCmd.SetHelpTemplate(rootCustomHelpTemplate)
 
 	rootCmd.PersistentFlags().BoolVar(&debugMode, "debug", false, "Run in debug mode")
 
-	// Manually parse flags to determine debug mode early
+	rootCmd.PersistentFlags().Bool("version", false, "Fetch the latest version")
+
+	// Manually parse flags to determine debug mode and version flag early
 	debugMode = checkForDebugFlag(os.Args[1:])
-	// Now that flags are parsed, set up the l722ogger
+	versionFlag := checkForVersionFlag(os.Args[1:])
+	if versionFlag {
+		// Fetch the version and print it
+		currentVersion := utils.KeployVersion
+		fmt.Println("Current version:", currentVersion)
```

Comment:
Print something like this:

Keploy xxx

not this current version : xxx and this print statement should come before that box present in the image.